src/main/java/no/finntech/maven/notifications/HipChat.java

```diff
@@ -75,18 +75,15 @@ protected void executeImpl() throws MojoExecutionException {
             throw new MojoExecutionException("\nhipchatMessage isn't defined.");
         }
 
-        if(!hipchatAnnouncement.exists()) {
-            throw new MojoExecutionException('\n' + hipchatAnnouncement.getAbsolutePath() + " doesn't exist.");
-        }
-
-
         final StringBuilder msg = new StringBuilder(hipchatMessage);
-        try {
-            msg.append("\n").append(new String(Files.readAllBytes(hipchatAnnouncement.toPath()), Charset.forName("UTF-8")));
-        } catch (IOException e) {
-            msg.append("failed to add release announcement. Due to: ").append(e.getMessage());
+        if(hipchatAnnouncement.exists()) {
+            try {
+                msg.append("\n").append(new String(Files.readAllBytes(hipchatAnnouncement.toPath()), Charset.forName("UTF-8")));
+            } catch (IOException e) {
+                msg.append("failed to add release announcement. Due to: ").append(e.getMessage());
+            }
```

Comment:
it would be nice here to detect when a hipchatAnnouncement has been configured but doesn't exist, that should still throw an exception.

but optional, as in simply not configured, should be ok and safely ignored.